applications/CompressiblePotentialFlowApplication/custom_elements/incompressible_potential_flow_element.cpp

```diff
@@ -744,33 +750,33 @@ void IncompressiblePotentialFlowElement<Dim, NumNodes>::ComputeVelocityLowerWake
 }
 
 template <int Dim, int NumNodes>
-double IncompressiblePotentialFlowElement<Dim, NumNodes>::ComputePressureUpper(const ProcessInfo& rCurrentProcessInfo) const
+double IncompressiblePotentialFlowElement<Dim, NumNodes>::ComputePressureUpper() const
 {
     const IncompressiblePotentialFlowElement& r_this = *this;
     const int wake = r_this.GetValue(WAKE);
 
     if (wake == 0)
-        return ComputePressureNormalElement(rCurrentProcessInfo);
+        return ComputePressureNormalElement();
     else
-        return ComputePressureUpperWakeElement(rCurrentProcessInfo);
+        return ComputePressureUpperWakeElement();
 }
 
 template <int Dim, int NumNodes>
-double IncompressiblePotentialFlowElement<Dim, NumNodes>::ComputePressureLower(const ProcessInfo& rCurrentProcessInfo) const
+double IncompressiblePotentialFlowElement<Dim, NumNodes>::ComputePressureLower() const
 {
     const IncompressiblePotentialFlowElement& r_this = *this;
     const int wake = r_this.GetValue(WAKE);
```

Comment:
Correct me if I'm wrong, but as far as I understand, this variable is used to mark which elements belong to the wake. If this is the case I'd suggest to use a flag.

Reply:
I also got his comment from @philbucher and @msandre in previous PRs. I remember we tried it but we had some issues, which to be honest right now I do not remember. I think it was something related to outputing flags in GiD, but I think there was something else too. We can try again when you are back and if you do not mind we will make this change in a future PR.